src/Castle.Windsor.Tests/MicroKernel/ArgumentsTestCase.cs

```diff
@@ -102,22 +102,6 @@ public void Custom_stores_get_picked_over_default_ones()
 			Assert.AreEqual(value, arguments["boo!"]);
 		}
 
-#if !SILVERLIGHT
-		[Test]
-		public void Custom_stores_get_picked_over_default_ones_in_clone()
-		{
-			var arguments = new Arguments(new CustomStringComparer());
-			var key = "foo";
-			var value = new object();
-
-			arguments.Add(key, value);
-
-			var clone = (IDictionary)((ICloneable)arguments).Clone();
```

Comment:
I thought this unit test would still work, just remove the cast to `ICloneable`?